test/e2e-common.sh

```diff
@@ -446,7 +446,7 @@ function knative_teardown() {
     done
   fi
   # Delete the test namespace
-  kubectl delete namespace ${E2E_SYSTEM_NAMESPACE} --ignore-not-found=true
+  add_trap "kubectl delete namespace ${E2E_SYSTEM_NAMESPACE} --ignore-not-found=true" SIGKILL SIGTERM SIGQUIT
```

Comment:
Move this next to the `create ns`, akin `defer mux.Unlock()` in Go.

Reply:
done.